### PR TITLE
add element-coder

### PR DESCRIPTION
**Pull request by kjappelbaum**
Checklist
- [x] Title of this PR is meaningful: e.g. "Adding my_nifty_package", not "updated meta.yaml".
- [x] License file is packaged (see [here](https://github.com/conda-forge/staged-recipes/blob/5eddbd7fc9d1502169089da06c3688d9759be978/recipes/example/meta.yaml#L64-L73) for an example).
- [x] Source is from official source.
- [x] Package does not vendor other packages. (If a package uses the source of another package, they should be separate packages or the licenses of all packages need to be packaged).
- [x] If static libraries are linked in, the license of the static library is packaged.
- [x] Package does not ship static libraries. If static libraries are needed, [follow CFEP-18](https://github.com/conda-forge/cfep/blob/main/cfep-18.md).
- [x] Build number is 0.
- [x] A tarball (`url`) rather than a repo (e.g. `git_url`) is used in your recipe (see [here](https://conda-forge.org/docs/maintainer/adding_pkgs.html#build-from-tarballs-not-repos) for more details).
- [x] GitHub users listed in the maintainer section have posted a comment confirming they are willing to be listed there.
- [x] When in trouble, please check our [knowledge base documentation](https://conda-forge.org/docs/maintainer/knowledge_base.html) before pinging a team.

### CODE REVIEW

**Comment by kjappelbaum** [comment]
I'm also willing to be a maintainer.

**Comment by kjappelbaum** [comment]
>Scirus not available on anaconda anywhere it seems
https://anaconda.org/search?q=Scirus

Yep, on it https://github.com/conda-forge/staged-recipes/pull/19636

**Comment by kjappelbaum** [comment]
@conda-forge/help-python ready for review!

File: recipes/element-coder/meta.yaml [hunk 1]
```diff
@@ -0,0 +1,49 @@
...
+  entry_points:
+    - element_coder.encode = element_coder.cli:encode_cli
+    - element_coder.decode = element_coder.cli:decode_cli
+
+
+
+requirements:
+  host:
+    - pip
+    - python >=3.7
```
Comment (BastianZim) [inline recipes/element-coder/meta.yaml:26]:
```suggestion
    - python >=3.7
    - setuptools >=35.0.2
    - setuptools_scm >=2.0.0,<3
```

File: recipes/element-coder/meta.yaml [hunk 2]
```diff
@@ -0,0 +1,49 @@
...
+    - pymatgen
+    - loguru
+    - click
+
+test:
+  imports:
+    - element_coder
+  commands:
+    - pip check
```
Comment (BastianZim) [inline recipes/element-coder/meta.yaml:37]:
```suggestion
    - pip check
    - element_coder.encode --help
    - element_coder.decode --help
```
This checks the CLI. Depends on whether you are actually offering a help function though.

**Review by BastianZim — CHANGES_REQUESTED** [review]
Just a couple of small things, then this is ready to go 🚀

**Comment by kjappelbaum** [comment]
@BastianZim thanks for the feedback, committed both your suggestions!

**Review by BastianZim — APPROVED** [review]

**Pull request merged**